dojo/listbox.m.css

```diff
@@ -36,5 +37,16 @@
 }
 
 .selectedOption {
+	composes: checkIcon;
```

Comment:
I think this can cause issues with duplicated class names and doesn't make it clear where you are composing from. Are you able to change to `composes checkIcon from './icons.m.css'` please? Also, are you missing `icon` class here too?

Reply:
It can't use the `icon` class, since only the pseudo-element should be an icon, and the actual element has normal text.